app/cypress/support/commands.js

```diff
@@ -34,8 +34,8 @@ Cypress.Commands.add('login', (username, password) => {
 });
 
 Cypress.Commands.add('logout', () => {
-  cy.clearCookie('mocks.auth');
-  cy.request('/logout');
+  cy.get('header').contains('Logout').click();
+  cy.url().should('equal', Cypress.config().baseUrl + '/');
```

Comment:
shouldn't we still clear the cookie?

Reply:
no, `cy.logout` is only used when using the real login. Cypress will clear its own cookies automatically when using the mock login